config/routes.yml

```diff
@@ -42,7 +42,7 @@ admin_link_block_delete:
     _legacy_controller: AdminLinkWidget
 
 admin_link_block_update_positions:
-  path: /link-widget/update-positions
+  path: /link-widget/update-positions/{hookId}
```

Comment:
Can we add some validation rule here, to make sure we handle a integer?

Reply:
right good idea! done ;)